setup.py

```diff
@@ -35,14 +35,14 @@
               'bin/ddg2pdb',
               'bin/ddg2summary'],
 
-    install_requires=[ 'biopython',
-                       'matplotlib',
-                       'numpy',
-                       'pandas',
-                       'logomaker',
-                       'scipy',
-                       'six',
-                       'openpyxl',
+    install_requires=[ 'biopython==1.84',
+                       'matplotlib>=3.9',
+                       'numpy>=2.1.3',
+                       'pandas>=2.2.3',
+                       'logomaker==0.8',
+                       'scipy>=1.14.1',
+                       'six>=1.14.1',
+                       'openpyxl>=3.1.5',
```

Comment:
what is the reasoning behind these version specification? Just for me to know where these come from

Reply:
Reason for these versions: 

I Installed mutatex in a new python 3.12 environment without versioning the the packs. Once fixing the update and the code, I checked which versions of the packs were used and wrote these here. These are all the newest stable version of the packs. 

Essentially we don't gain anything from having the versions listed at the moment. Only when updates happen such as the biopython or matplotlib changes affects the utility of MutateX does it make sense to list specific versions. 

Suggestion - we drop these requirements, so the user simply download the newest versions, and instead leave a notice in the manual or .md that per 9th of December 2024 MutateX was updated to these versions of the packs, so the user can troubleshoot themself, open an issue but know the version to revert to. We can do so again in the future if we do a similar update.

Reply:
Updated as per agreement.